src/main/kotlin/org/coup/game/actions/Player.kt

```diff
@@ -1,14 +1,48 @@
 package org.coup.game.actions
 
-class Player(private val name: String) {
+/**
+ * This class represents a player in the game.
+ *
+ * @param name The name of the player.
+ * @property coins The number of coins the player has.
+ * @property influence The influence the player has.
+ * @constructor Creates a new player.
+ */
+data class Player(val name: String) {
     var coins: Int = 0
         private set
+    var influence: Int = 0
```

Comment:
Só um comentário mesmo. `influence` aqui é um inteiro, mas muito provavelmente vamos precisar adicionar de fato uma referência de qual é a influência que o Player possui. Vai ser útil para quando implementar as ações de que o player precisa provar que ele de fato tem a influência.

Reply:
Sim, é a ideia mesmo. Só deixei inteiro pra não me preocupar com modelar as influencias por agora.

É o que o comentário no PR: `Added the concept of influences only to help the flow. This will be changed in the future.` tenta explicar.